frontend/src/store/NotificationState.ts

```diff
@@ -3,27 +3,78 @@
     It is structured this way so it doesn't stack multiple messages with the same severity (by design).
     It is intended to be used in conjunction with the NotificationStack component.
 */
-import type { Severity } from "@/enum/SeverityEnum";
+
 import { ref } from "vue"
+import { isDesktop, screenSize } from "@/store/screenSizeState";
+import type { Severity } from "@/enum/SeverityEnum";
 
 const defaultNotification = {
     success: "",
     warn: "",
     error: "",
 };
 
+export const newMsgList = ref();
+export const newFormData = ref();
+export const newSeverity = ref();
+
 export const notifications = ref(
     JSON.parse(JSON.stringify(defaultNotification))
 );
 
 export const pushNotification = (severity: Severity, message: string) => {
     notifications.value[severity] = message;
-}
+};
 
 export const clearNotification = (severity: string) => {
     notifications.value[severity] = '';
-}
+};
 
 export const resetNotification = () => {
     notifications.value = JSON.parse(JSON.stringify(defaultNotification));
-}
+};
+
+//how many full id will be shown per message
+const idPerMsg = () => {
+    if(screenSize <= 725) {
+        return 1;
+
+    } else if(screenSize <= 1060 && isDesktop() || screenSize <= 820) {
+        return 2;
+
+    } else if(screenSize <= 1130 && isDesktop() || screenSize <= 880) {
+        return 3;
+
+    } else if(screenSize <= 1290 && isDesktop() || screenSize <= 960) {
+        return 4;
+
+    } else {
+        return 5;
+
+    }
+};
```

Comment:
The implementation requirement isn't clear here.
Did you discuss with business (or @OlgaLiber2 / @MCatherine1994 ) in the ticket that: hiding id(s) for the user is **acceptable**? 
Is it one of the acceptance criteria in the ticket? Please add this if its been discussed.
(We don't just make code works).
We need to ask, for example:
* how many of the ids can the field take, is there such requirement?
* if there is no limit, can we fit the ids in the rendering container conditionally (css style change)?
* if there is no limit, for mobile device, what else can we do to handle it better before we really chop the ids.
Please discuss these kind of questions before implementing it or before opening a PR if you already see a problem.

Reply:
hiding the ids that don't fit is on the design